app/Libraries/Api/Models/BaseApiModel.php

```diff
@@ -696,6 +696,8 @@ protected function castAttribute(string $key, mixed $value): mixed
                 return new BaseCollection($this->fromJson($value));
             case 'datetime':
                 return $this->asDateTime($value);
+            case 'date':
+                return $this->asDateTime($value)->toDateString();
```

Comment:
My only concern with this change is that the `aic_start_at` and `aic_end_at` attributes used to return a `Carbon` object, but now they return a string. @zachgarwood, would this have a negative impact elsewhere in the app?

Reply:
I don't think so, we're not doing any comparisons on those dates, so I don't think it will matter.